android/anc/src/main/java/org/smartregister/fhircore/anc/data/family/FamilyDetailRepository.kt

```diff
@@ -17,12 +17,14 @@
 package org.smartregister.fhircore.anc.data.family
 
 import com.google.android.fhir.FhirEngine
+import com.google.android.fhir.search.Search
```

Comment:
do we need this and the below line?

Reply:
yeah, for the Search class is used on line 47. compiler complains otherwise